frontstage/controllers/conversation_controller.py

```diff
@@ -257,6 +266,13 @@ def _create_survey_options(respondent_enrolments: dict) -> list:
     return survey_options
 
 
+def _create_business_options(business_selection: dict) -> list:
+    business_options = [Option("Not business related", "Not business related")]
```

Comment:
I couldn't see this option in any of the docs, is it a requirement to have a Not business related option?

Reply:
Removed.